requests_toolbelt/utils/dump.py

```diff
@@ -104,7 +104,10 @@ def _dump_response_data(response, prefixes, bytearr):
 
     bytearr.extend(prefix + b'\r\n')
 
-    bytearr.extend(response.content)
+    if not response._content_consumed or response._content is False:
```

Comment:
I think we avoid private attributes everywhere else in the code base. I wonder if something a little simpler is easier:

```py
>>> r = requests.get("https://httpbin.org/get")  
>>> r.raw.closed
True
>>> s = requests.get('https://httpbin.org/get', stream=True)
>>> s.raw.closed
False
```

Reply:
Good catch, that seems to be working just as well. I updated the PR and added a test